packages/scaffold/src/partials/w3m-account-wallet-features-widget/index.ts

```diff
@@ -139,6 +142,24 @@ export class W3mAccountWalletFeaturesWidget extends LitElement {
     return html`<wui-balance dollars="0" pennies="00"></wui-balance>`
   }
 
+  private activateAccountTemplate() {
+    const smartAccountEnabled = NetworkController.checkIfSmartAccountEnabled()
+    const preferredAccountType = W3mFrameHelpers.getPreferredAccountType()
+    if (
+      !smartAccountEnabled ||
+      preferredAccountType === 'smartAccount' ||
+      this.smartAccountDeployed
+    ) {
+      return null
+    }
+
+    return html` <wui-promo
+      text=${'Activate your account'}
+      @click=${() => RouterController.push('UpgradeToSmartAccount')}
```

Comment:
Let's not use inline function definitions, instead we could define the onClick method outside as private method:

```js
private onUpdateToSmartAccount () {
   RouterController.push('UpgradeToSmartAccount')
}
..
@click=${this.onUpdateToSmartAccount.bind(this)}
`